src/TLuaInterpreter.cpp

```diff
@@ -106,7 +106,7 @@ TLuaInterpreter::TLuaInterpreter( Host * pH, int id )
     connect(&purgeTimer, SIGNAL(timeout()), this, SLOT(slotPurge()));
 
     mpFileDownloader = new QNetworkAccessManager( this );
-    connect(mpFileDownloader, SIGNAL(finished(QNetworkReply*)), this, SLOT(replyFinished(QNetworkReply*)));
+    connect(mpFileDownloader, SIGNAL(finished(QNetworkReply*)), this, SLOT(slot_replyFinished(QNetworkReply*)));
```

Comment:
I like slot (and signals) to be marked out - it helps to remember that they can get called from outside the class at odd-times (asynchronously ?)